package.json

```diff
@@ -9,7 +9,8 @@
     "test": "standard && (nyc -s tape test/*-test.js | faucet) && nyc report",
     "test-gc": "npx -n=--expose-gc tape test/{cleanup,iterator-gc,chained-batch-gc}*-test.js",
     "test-electron": "electron test/electron.js",
-    "test-prebuild": "cross-env PREBUILDS_ONLY=1 npm t",
+    "test-prebuild": "cross-env PREBUILDS_ONLY=1 tape test/*-test.js | faucet",
```

Comment:
Skipping `standard` and `nyc` because `standard` hits some permission issue, and without these we can use a readonly volume (`$(pwd):/app:ro`).